docs_src/pipelinesetup.md

```diff
@@ -33,11 +33,9 @@ sh modelDownload.sh
 
 You must build the provided component services and create local docker images. The following table lists the build command for various platforms. Choose and run the command corresponding to your platforms or hardware.
 
-| Platform                                   | Docker Build Command       | Check Success                                     |
-| ------------------------------------------ | -------------------------- |---------------------------------------------------|
-| Intel platforms with Intel integrated GPUs | <pre>make build-soc</pre>  | docker images command to show <b>sco-soc:2.0</b>  |
-| Intel platforms with Intel discrete GPUs   | <pre>make build-dgpu</pre> | docker images command to show <b>sco-dgpu:2.0</b> |
-| build both platforms                       | <pre>make build-all</pre>  | docker images command to show both above          |
+| Platform                                   | Docker Build Command       | Check Success                                                |
+| ------------------------------------------ | -------------------------- |--------------------------------------------------------------|
+| Intel platforms using DLStreamer           | <pre>make build-dlstreamer</pre>  | docker images command to show <b>sdlstreamer:2.0</b>  |
```

Comment:
should we also include the build-dlstreamer-realsense? it is worth documentation here too so for realsense users they can build with it